### DISCUSSION TITLE
TypeScript 4.2 includes ResizeObserver interfaces 🎉

### DISCUSSION THREAD

**Discussion by mdingena** [juggle/resize-observer#122] [General]
It looks like everything made it into TS 4.2 except `pixelDeviceContentBoxSize`.

With that in mind, you are probably able to clean up a lot of your own types! 🚀

**Comment by TremayneChrist** [comment]
Ah that's cool! Thanks for the info Marc! I guess I'll need them there for backwards compatibility, though 🤔
